Bottom: a8166968261d2b2f3fdf2e2f7f9ee43742b49d7c
Top:    d20aa268d640b43da7c31e73cd8f49c448c0bc0a
Author: Lawrence D'Anna <lawrence_danna@apple.com>
Date:   2019-09-30 17:03:19 -0700

Refresh of new-api-class-sbfile

---
diff --git a/lldb/packages/Python/lldbsuite/test/python_api/file_handle/TestFileHandle.py b/lldb/packages/Python/lldbsuite/test/python_api/file_handle/TestFileHandle.py
index 76a8441df5d..29b0320e062 100644
--- a/lldb/packages/Python/lldbsuite/test/python_api/file_handle/TestFileHandle.py
+++ b/lldb/packages/Python/lldbsuite/test/python_api/file_handle/TestFileHandle.py
@@ -81,6 +81,7 @@ class FileHandleTestCase(lldbtest.TestBase):
             # even with collect_result=True.
             self.handleCmd('script 1+1')
             self.debugger.GetOutputFileHandle().write('FOO\n')
+        lldb.SBDebugger.Destroy(self.debugger)
         with open(self.out_filename, 'r') as f:
             self.assertEqual(readStrippedLines(f), ['2', 'FOO'])
